client_response.go

```diff
@@ -29,18 +29,25 @@ type ResponseMetadata struct {
 	StatusCode int
 	// Version indicates the response version
 	Version string
+	// TotalRecords is the header corresponding to the `Recurly-Total-Records`. The count of records matching this pager.
+	TotalRecords *int64
 	// Request is the metadata describing the request for this response
 	Request RequestMetadata
 }
 
 func parseResponseMetadata(res *http.Response) *ResponseMetadata {
 	deprecated := res.Header.Get("Recurly-Deprecated") == "TRUE"
+	totalStr := res.Header.Get("Recurly-Total-Records")
+	total, _ := strconv.ParseInt(totalStr, 10, 64)
+	// if err != nil {
```

Comment:
Probably want to set TotalRecords to `nil` if it doesn't parse or doesn't apply.